src/entry.js

```diff
@@ -12,6 +13,10 @@ function StacheEntryPlugin() {
         require('./template-reference-variable')
       ];
 
+      if (!stacheJsonDataService.getStacheDataObject()) {
```

Comment:
I'm struggling to make sense of this condition being in this context.

Why not run this check every time `getStacheDataObject()` is called? This way, the implementation details will be hidden from the consumer of the JSON data utility (all the consumer wants to do is get the data object; he shouldn't have to worry about setting it first).